src/fasta-recs.py

```diff
@@ -11,8 +10,22 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # print(f"Now I need to process the records in {args.fasta}")
+
+    printer = ''
+    for line in (args.fasta):
+        if '>' in line:
+            if printer != '':
+                print(printer)
+                printer = ''
+            name = line.replace('>',  '')
```

Comment:
This removes *all* `>` in `line`. If the name contained `>` you shouldn't remove that. You should only remove the first `>`. If you only enter this branch when `line.startswith('>')` it is safe to do `name = line[1:]`.